### PR TITLE
DI-95 - Findability subset

### PR DESCRIPTION
**Pull request by ncalvanese1**
Hosting the FSS schema here for the time being

### CODE REVIEW

File: scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+    },
+    {
+      "name": "biosample",
+      "columns": [
+        {"name": "biosample_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "anatomical_site", "datatype": "string", "array_of": false},
+        {"name": "apriori_cell_type", "datatype": "string", "array_of": true},
+        {"name": "biosample_type", "datatype": "string", "array_of": false},
+        {"name": "diagnosis_id", "datatype": "string", "array_of": true},
+        {"name": "disease", "datatype": "string", "array_of": false},
```
Comment (mbaumann-broad) [inline scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json:66]: I would think a biosample could be associated with more than one disease, could it not?
Wondering if this should be an array.

File: scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    {
+      "name": "activity",
+      "columns": [
+        {"name": "activity_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "activity_type", "datatype": "string", "array_of": false},
+        {"name": "used_file_id", "datatype": "string", "array_of": true},
+        {"name": "generated_file_id", "datatype": "string", "array_of": true}
+      ],
+      "primaryKey": ["activity_id"],
+      "partitionMode": "none",
```
Comment (mbaumann-broad) [inline scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json:12]: What are `partitionMode`, `datePartitionOptions` and `intPartitionOptions` used for?
Are these hints to the Azul indexer to aid efficient parallel indexing, or something else?

File: scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+    },
+    {
+      "name": "file",
+      "columns": [
+        {"name": "file_id", "datatype": "string", "array_of": false, "required": true},
+        {"name": "data_modality", "datatype": "string", "array_of": true},
+        {"name": "file_format", "datatype": "string", "array_of": false},
+        {"name": "file_ref", "datatype": "fileref", "array_of": false},
+        {"name": "reference_assembly", "datatype": "string", "array_of": true},
+        {"name": "label", "datatype": "string", "array_of": false}
```
Comment (mbaumann-broad) [inline scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json:138]: I think there should also be attributes for file size and file checksums.
FYI, the BioData Catalyst minimal metadata model for files is available here for reference/comparison:
https://docs.google.com/document/d/1U3uh4P8oHKDvKTwi5HSwmije0q0MuxiSp-i3-6cSqIw/edit

File: scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json (hunk 2 above)
Comment (ncalvanese1) [reply scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json:12]: These are used by TDR as instructions on how to partition the data in BigQuery.

File: scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json (hunk 1 above)
Comment (ncalvanese1) [reply scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json:66]: I think this is a really good question. I've added to https://broadworkbench.atlassian.net/browse/DI-121 (where I'm consolidating potential FSS schema updates).

File: scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json (hunk 3 above)
Comment (ncalvanese1) [reply scripts/anvil_tdr_ingest/findability_subset/findability_subset_schema.json:138]: The "findability" naming convention we are using is perhaps a bit misleading. Essentially, this schema should encompass anything we want to capture consistently across contributing datasets. I agree that these two items are potentially valuable to capture and have added them to https://broadworkbench.atlassian.net/browse/DI-121 (where I'm consolidating potential updates to the FSS schema).

**Pull request closed without merging**